plugins/mediatek-scaler/mediatek-scaler.quirk

```diff
@@ -1,8 +1,2 @@
-[PCI\VID_1028&PID_0C88]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0C8A]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0CE9]
-Flags = probe-vcp
-[PCI\VID_1028&PID_0CEB]
-Flags = probe-vcp
+[DRM\VEN_DEL&DEV_4340]
+Plugin = mediatek_scaler
```

Comment:
@CragW any more  needed here?

Reply:
Yes, we need some additions. Thanks for checking.
```
[DRM\VEN_DEL&DEV_7430]
Plugin = mediatek_scaler
```